### PR TITLE
UI for profile : theme settings

### PR DESCRIPTION
**Pull request by Mai-LinhC**
- [x] Change name of page to preferences
- [x] Theme
- [x] Text size
- [x] Language
- [x] Currency
- [x] Test

### CODE REVIEW

File: app/src/main/java/com/github/se/assocify/ui/composables/DropdownWithSetOptions.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+
+/**
+ * A dropdown with a list of given options, readOnly text field
+ *
+ * @param options: The list of options
```
Comment (sarahbadr17) [inline app/src/main/java/com/github/se/assocify/ui/composables/DropdownWithSetOptions.kt:16]: miss modifier parameters

File: app/src/main/java/com/github/se/assocify/ui/composables/DropdownWithSetOptions.kt [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                  onSelectOption(item)
+                  onOpenedChange(false)
+                },
+                leadingIcon = leadIcon,
+                modifier = Modifier.testTag("DropdownItem-${item.uid}"))
+          }
+        }
+      }
+}
+
```
Comment (sarahbadr17) [inline app/src/main/java/com/github/se/assocify/ui/composables/DropdownWithSetOptions.kt:59]: add comments

File: app/src/main/java/com/github/se/assocify/ui/screens/profile/preferences/ProfilePreferencesScreen.kt [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+          "Chinese",
+          "Japanese",
+          "Korean",
+          "Arabic",
+          "Hindi",
+          "Turkish",
+          "Dutch",
+          "Polish",
+          "Swedish",
+          "Danish",
```
Comment (sarahbadr17) [inline app/src/main/java/com/github/se/assocify/ui/screens/profile/preferences/ProfilePreferencesScreen.kt:72]: Didn't you forget some languages? :) Can't you fetch this list from somewhere? Maybe an API or something?

**Review by sarahbadr17 — COMMENTED** [review]
Good job ! Just some little comments :)

File: app/src/main/java/com/github/se/assocify/ui/screens/profile/preferences/ProfilePreferencesScreen.kt (hunk 3 above)
Comment (Mai-LinhC) [reply app/src/main/java/com/github/se/assocify/ui/screens/profile/preferences/ProfilePreferencesScreen.kt:72]: Made me think about whether we want to implement languages or not : I don't believe we do for v1 of the app.
I therefore disabled the language option :)

File: app/src/main/java/com/github/se/assocify/ui/screens/profile/preferences/ProfilePreferencesScreen.kt [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+@Composable
+fun ProfilePreferencesScreen(navActions: NavigationActions) {
+  // temporary values for the theme, text size, language and currency, waiting for the viewmodel
+  val themeOptions = listOf("Light", "Dark", "System")
+  var themeSelectedIndex by remember { mutableIntStateOf(0) }
+
+  var sliderPosition by remember { mutableFloatStateOf(15f) }
+
+  var openLanguageDropdown by remember { mutableStateOf(false) }
+  val languageOptions = listOf(DropdownOption("English", "English"))
```
Comment (sarahbadr17) [inline app/src/main/java/com/github/se/assocify/ui/screens/profile/preferences/ProfilePreferencesScreen.kt:55]: Shouldn't it be English, French?

**Review by sarahbadr17 — APPROVED** [review]
LGTM! Nice job

**Pull request merged**